pkg/appcontext/context.go

```diff
@@ -67,3 +69,14 @@ func Principal(c context.Context) authn.Principal {
 	}
 	return authn.ANON
 }
+
+// WithLDAppEnvUser decorates the context with the app env user
+func WithLDAppEnvUser(c context.Context, u ld.User) context.Context {
+	return context.WithValue(c, ldUserKey, u)
+}
+
+// LDAppEnvUser returns the default user for the LD Api
+func LDAppEnvUser(c context.Context) (ld.User, bool) {
```

Comment:
I have proposed to eliminate these accessors in a different comment.

But if not eliminating them, I have a stylistic proposal for this accessor. Instead of having a conditional return (`bool`), why not ALWAYS return an `ld.User`... Either the found one, or if not found return an Anonymous representation.